.gitignore

```diff
@@ -13,18 +13,16 @@ lib-cov
 # Coverage directory used by tools like istanbul
 coverage
 
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
 # node-waf configuration
 .lock-wscript
 
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
 
 # secret stuff
-secret
+.env
+
+# dist dir
+dist
```

Comment:
Top level ignore makes sense. Yea I think this was copied from the github gitignore templates repo, so it includes some stuff we don't use.